Misc/NEWS.d/next/Core and Builtins/2022-03-08-10-50-42.bpo-46944.cnaIK3.rst

```diff
@@ -0,0 +1 @@
+Speed up throwing exception in generator by 20%. Patch by Kumar Aditya.
```

Comment:
```suggestion
Update .generator.throw to use FASTCALL calling convention Patch by Kumar Aditya.
```

Performance gains may be relative results depending on the environment. I prefer neutral comments.